src/menu.cpp

```diff
@@ -219,33 +217,6 @@ namespace App
                     LOG_S(INFO) << "> " << commands.back();
                     redraw = true;
 
-                    PyObject *m, *d, *v;
-                    m = PyImport_AddModule("__main__");
-
-                    if (m == NULL)
-                        break;
-
-                    d = PyModule_GetDict(m);
-                    v = PyRun_String(commands.back().c_str(), Py_single_input, d, d);
```

Comment:
This means that the console doesn't work anymore. Please hook up lua or angelscript!